Update processes.md (#3368)
viamrobotics/docs: docs/configure/processes.md

1. [`ssh` into your board](/installation/prepare/rpi-setup/#connect-with-ssh) and install `pip`:

   ```sh {class="command-line" data-prompt="$"}
   sudo apt install -y python3-pip
   ```

2. Install the Viam Python SDK (and other dependencies if required) into a new folder called `robot`:

   ```sh {class="command-line" data-prompt="$"}
   pip3 install --target=robot viam-sdk <other-required-dependencies>
   ```

3. Display the full path of the current directory you are working in on your board. Make a note of this output for the next step.

   ```sh {class="command-line" data-prompt="$"}
   pwd
   ```

> 4. From your computer (not the ssh prompt to your board), add your code to your new folder. In the command, you will copy `main.py` over to your board, with the section following the colon `:` indicating where your file should be copied to on the board (the path of the directory you are working in on your board, along with the filename).

Reviewer: I don't actually think we want to be that specific, for all we know they've got a keyboard and monitor connected to their board... 

```suggestion
4. Add your code to the <file>robot</file> folder.
```

>    ```sh {class="command-line" data-prompt="$"}
>    scp main.py user@host.local:/home/myboard/robot/main.py

Reviewer: ```suggestion
```

1. [`ssh` into your board](/installation/prepare/rpi-setup/#connect-with-ssh) and install `pip`:

   ```sh {class="command-line" data-prompt="$"}
   sudo apt install -y python3-pip
   ```

2. Install the Viam Python SDK (and other dependencies if required) into a new folder called `robot`:

   ```sh {class="command-line" data-prompt="$"}
   pip3 install --target=robot viam-sdk <other-required-dependencies>
   ```

3. Display the full path of the current directory you are working in on your board. Make a note of this output for the next step.

   ```sh {class="command-line" data-prompt="$"}
   pwd
   ```

4. Add your code to the <file>robot</file> folder.

>    ```

Reviewer: ```suggestion
```

1. [`ssh` into your board](/installation/prepare/rpi-setup/#connect-with-ssh) and install `pip`:

   ```sh {class="command-line" data-prompt="$"}
   sudo apt install -y python3-pip
   ```

2. Install the Viam Python SDK (and other dependencies if required) into a new folder called `robot`:

   ```sh {class="command-line" data-prompt="$"}
   pip3 install --target=robot viam-sdk <other-required-dependencies>
   ```

3. Display the full path of the current directory you are working in on your board. Make a note of this output for the next step.

   ```sh {class="command-line" data-prompt="$"}
   pwd
   ```

4. Add your code to the <file>robot</file> folder.

> 

Reviewer: ```suggestion
```

<!-- prettier-ignore -->
| Attribute (Builder Mode) | Attribute (JSON) | Type    | Required? | Description |
| ------------------------ | ---------------- | ------- | --------- | ----------- |

> | Executable               | `name`               | string  | **Required** | The command you want to execute when your machine connects to the server. You can find the executable path of Python3 with `which Python3`. |

Reviewer: ```suggestion
| Executable               | `name`               | string  | **Required** | The command you want to execute when your machine connects to the server. On many operating systems, you can find the executable path of commands by running with `which <command-name>`. |
```
